target/linux/ramips/dts/YOUKU-YK2.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "youku,yk2", "mediatek,mt7621-soc";
+	model = "YOUKU YK2";
+
+	aliases {
```

Comment:
led_power can be used as led-running and led-upgrade as well. You could add them here.

Reply:
ok , will add it.